src/config/Config.py

```diff
@@ -29,53 +32,76 @@ def ingest_env(self):
         if "PIPELINE_DB_NAME" in os.environ:
             self._data["pipeline_db_name"] = os.environ["PIPELINE_DB_NAME"]
 
+    def _is_date(self, val):
+        if not isinstance(val, str):
+            return False
+
+        if len(val) == 10:
+            for i in range(len(val)):
+                if i == 4 or i == 7:
+                    if val[i] != '-':
+                        return False      
+                else:
+                    if not val[i].isnumeric():
+                        return False
+        else:
+            return False                
+        return True
+
+    def _is_na(self, val):
+        if val == 'NA' or val == 'na' or val == '':
```

Comment:
What about N/A? :)

Reply:
Or 'n/a', if that isn't implied from Tin's comment.